lec01.tex

```diff
@@ -146,8 +146,8 @@ \subsection{Equivalent Definitions of prBPP via Amplification}
 \begin{definition}
   An \emph{$(\eps, \delta)$-sampler} is a function $s :
   \set{0,1}^{\overline{T}} \times \set{0,1}^\ell \to \set{0,1}^T$ such that for
-  all $f : \set{0,1}^T \to \set{0,1}$ and for all $z \in
```

Comment:
Iirc, and on my notes, it's probabilistic over $z$.

Reply:
And with the current way it's written, the $\delta$ in $(\eps, \delta)$ seems superfluous.